### ISSUE TITLE
There are some problems with the new version 0.14-p3

### ISSUE THREAD

**Issue by Kotarou2k4** [AOF-Dev/MCinaBox#423]
Here it some problems, resson and also some suggestions:
1) MCPE Item bar some time stuck in top left corner screen
2) I can't change the buttons background color in custom keyboard, can't press out the dots when changing margins, width and height
3) The setting wrench icon hide my fps and I can't set buttons in the corner below, it alway change to the higher postion (with picture below)
![PicsArt_09-29-10 17 05](https://user-images.githubusercontent.com/65490046/94508924-49d6b300-023d-11eb-8f05-35bbc64662bd.jpg)

4) Can you re-add the color picker in custom keyboard editor ?
5) Add a feature that can save the postion of virtual controller together with custom keyboard because if we re download the app, we need to edit the postion again. I want to use Mcpe keyboard because it can swipe like on Mcpe version so it moving more flexibly
6) Add an option in Touchpad setting that will disable the mouse buttons (working like old Boat app version, need to press mouse buttons) because some time when I accidentally miss press the button and it trigger the right mouse button, that can be a big problem when play with some mod

I hope you understand it

**Comment by Kotarou2k4** [comment]
@longjunyu2

**Comment by longjunyu2** [comment]
Hello @Kotarou2k4 , this is longjunyu2.
I think you didnot explore the whole functions about MCinaBox0.1.4
I will respond to you later(I have a course.)

**Comment by rApHaEL1211** [comment]
My mcinabox crash I download ver 1.8.9 and then click start it crash plss help me

**Comment by rApHaEL1211** [comment]
@longjunyu2 can you help me when I download ver 1.8.9 and click start,it crash

**Comment by rApHaEL1211** [comment]
@longjunyu2 please

**Comment by Kotarou2k4** [comment]
@longjunyu2 i will be explore more but do you agree with some my suggestions ?

**Comment by rApHaEL1211** [comment]
Please I really want to play mcinabox

**Comment by rApHaEL1211** [comment]
:(

**Comment by longjunyu2** [comment]
I am gold to see your suggestions. But the suggestions 1,3,6 already achieved in 0.1.4

**Comment by longjunyu2** [comment]
If you want to automatically change the position of MCPE itembar, you just need preesed the refresh button which is in Virtual Controller.
I have got your suggestions.
will try to fix them.

**Comment by longjunyu2** [comment]
> Please I really want to play mcinabox

You did not upload anything helpful to find the bugs...
I want to help you but I just don not how to help you.

**Comment by longjunyu2** [comment]
@Kotarou2k4 you can drag the virtualcontroller icon. This will.change its position.
I will change its original position to the center left of the screen also.

**Comment by rApHaEL1211** [comment]
> > Please I really want to play mcinabox
> 
> You did not upload anything helpful to find the bugs...
> I want to help you but I just don not how to help you.
> 

Oh do I need to upload some bugs???

**Comment by rApHaEL1211** [comment]
How can I find the bugs and crash?

**Comment by Kotarou2k4** [comment]
@longjunyu2 No, you don't understand what I mean, I know I can change the postion. But I want that virtual control can be able to save the postion in custom keyboard file

**Comment by rApHaEL1211** [comment]
> > Please I really want to play mcinabox
> 
> You did not upload anything helpful to find the bugs...
> I want to help you but I just don not how to help you.
> 

It's ok if you can't help me I'm gonna figure it out myself😀😀

**Comment by longjunyu2** [comment]
> @longjunyu2 No, you don't understand what I mean, I know I can change the postion. But I want that virtual control can be able to save the postion in custom keyboard file

Get it. But this is hard to achieve because of present struct of codes...
OK, I will try.
Maybe not in MCinaBox0.1.4.

**Comment by longjunyu2** [comment]
I got a lot of bug reports and suggestions, I can not finished all of them in just few uodates.
please...

**Comment by Kotarou2k4** [comment]
Don't worry I will wait for it, you are try your best. I only hope you accept my suggestions in some later versions

**Comment by rApHaEL1211** [comment]
How can I send pic or vid so that I can show you what type of crash

**Comment by rApHaEL1211** [comment]
@longjunyu2 
[crash-2020-09-29_05.40.22-client.txt](https://github.com/AOF-Dev/MCinaBox/files/5296694/crash-2020-09-29_05.40.22-client.txt)

**Comment by AyHa1810** [comment]
@longjunyu2 the language of the "home" and the second section in the settings does not gets changed even if you change the language. Fix it.
![Screenshot_2020-09-29-21-57-39-12_4cb6582af5c3b01ac644d295dc92f43e](https://user-images.githubusercontent.com/63386848/94586807-50961200-029f-11eb-955d-d5c4d6f69df5.jpg)

**Comment by longjunyu2** [comment]
> @longjunyu2 the language of the "home" and the second section in the settings does not gets changed even if you change the language. Fix it.
> ![Screenshot_2020-09-29-21-57-39-12_4cb6582af5c3b01ac644d295dc92f43e](https://user-images.githubusercontent.com/63386848/94586807-50961200-029f-11eb-955d-d5c4d6f69df5.jpg)

OK. It seems that "主页" is hardcode.
